lambdas/package.json

```diff
@@ -1,5 +1,16 @@
 {
```

Comment:
`package-lock.json` changes shouldn't be needed as you haven't installed any new packages, and `npm ci` runs correctly.

Reply:
The lock files needs to be commited to the repo, though. You shouldn't get a diff when running `npm install` locally - it means some dependencies are out-of-date.